python/ray/data/_internal/logical/operators/read_operator.py

```diff
@@ -30,4 +34,4 @@ def fusable(self) -> bool:
         as fusion would prevent the blocks from being dispatched to multiple processes
         for parallel processing in downstream operators.
         """
-        return self._estimated_num_blocks == len(self._read_tasks)
+        return self._parallelism == self._estimated_num_blocks
```

Comment:
just `self._additional_split_factor is None`?

Reply:
if `_additional_split_factor == 1`, we can still do the fusion right?